docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,87 @@
++ 
+** Verify that the case designer includes the `Milestones` and `Cases` objects by either checking the *Object Library* dock on the left side of the case designer, or by checking the `WorkDefinition` file under *Work Item Definitions* in the *Project Explorer* window on the left side of the {CENTRAL} workbench.
+. *Define process level and case level variables.* 
++
+Click the image:3140.png[] arrow on the right side of the designer to open the `Properties (BPMN-Diagram)` panel. Click the down arrow next to `Variable Definitions` to open the `Editor for Variable Definitions`.
++
+.IT Orders variables
+image::case-variables.png[Editor for IT Orders variables]
++
+In the IT Orders sample project, you can see that certain variables are identified as `Case File`. A case file variable is stored in the case file itself, and promotes reuse of the case file data rather than copying it across multiple process instances. For example, the `hwSpec` variable is available to all processes within the case, and is accessible directly from the case file even without an existing process instance. 
++
+For more information about the case file, see <<case-management-case-file-con-{context}>>.
+
+. *Define the roles involved in the case.*
++
+Open the properties panel and click the down arrow next to `Variable Definitions` to open the `Editor for Variable Definitions`.For more information about roles, see <<case-management-roles-con-{context}>>.
+
+. *Define what work item will be triggered when a new case starts.*
++
+As case definition is an ad hoc process definition, it does not require an explicit start node. Nodes that have no incoming connections and are marked as `Adhoc Autostart` and are automatically triggered when an instance is started.
++
+By default, the case definition designer includes a `Milestone` object configured as `Adhoc autostart` in the palette. Modify this object or replace it with another work item, such as a user task or script task to be triggered when a new case is started.
++
+In the IT Orders example, the `Prepare hardware spec` user task and the `Milestone 1: Order placed` milestones are `Adhoc autostart`, and are automatically triggered when an instance is started by a request being made for new IT hardware.
+
+. *Define the `Place order` subprocess.*
++
+In the IT Orders example, the `place-order` process is a separate business process that is carried out by the supplier. This is a reusable process that occurs during the course of case execution. 
+.. From the project menu, click *Add Asset* -> *Business Process* to create a new business process. Define the name and package for the new process and click *Ok*. 
+.. Define the process that is a subprocess within the case. For example, the IT Orders `Place order` process is a simple user task that applies to the supplier group and receives the relevant order data and case variables directly from the case file.
+
+. *Define milestones*
```

Comment:
In milestones, it is probably useful to describe how you can evaluate a condition based on case file variables, including an example like: _org.kie.api.runtime.process.CaseData(data.get("ordered") == true)_ and step by step of how to include this in the design.

Reply:
Added to this procedure.